utoipa-gen/src/component/schema.rs

```diff
@@ -619,6 +619,14 @@ impl ToTokensDiagnostics for UnnamedStructSchema<'_> {
 
             if fields_len == 1 {
                 if let Some(ref mut features) = unnamed_struct_features {
+                    let inline =
+                        features::parse_schema_features_with(&first_field.attrs, |input| {
```

Comment:
@juhaku that looks kinda naive. Isn't there a way to properly specify which features are supported by this specific case?

Reply:
What I mean is I'd rather have something like this: https://github.com/juhaku/utoipa/blob/d020f92a0446adc7e99999aa9e1be63e95728fbb/utoipa-gen/src/component/schema/features.rs#L150

But those features are for the variant themselves, so `Inline` would be:

```rust
    #[derive(ToSchema)]
    enum Card {
        #[schema(inline)]
        Number(Number),
        #[schema(inline)]
        Color(Color),
    }
```

Which IMO does not fit with the rest of the API and is counter-intuitive.

But I haven't found the features struct for the unnamed field.

Reply:
> @juhaku that looks kinda naive. Isn't there a way to properly specify which features are supported by this specific case?

No features have been originally defined for unnamed field enum variants itself thus there is no such struct. Though if there is just a single feature to be parsed the `parse_with` is just fine but if there would be multiple features or the set of features is known to be needed in multiple locations then a feature struct is preferred. 

> Which IMO does not fit with the rest of the API and is counter-intuitive.


Yeah, I agree the API would be counter intuitive and the features defined directly for the unnamed enum variant field is cleaner approach. 

> But I haven't found the features struct for the unnamed field.

Such struct does not exists by design but could be added if needed.